chart/values.yaml

```diff
@@ -22,7 +22,7 @@ ingressMode: hosts
 certificatesSecret:
   secretName: https-certificates
 
-imagePrefix: gcr.io/gitpod-io/
+imagePrefix: gcr.io/gitpod-io/self-hosted
```

Comment:
the `imagePrefix` must end with a `/` ... it's not added when the prefix and name are joined.